src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Lottery {
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber((int)Math.floor(Math.random() * (100 - 0 + 1) + 0));
```

Comment:
```suggestion
        ball.setNumber((int) (Math.random() * (100 + 1)));
```
please make 100 a constant (constant is a static final field with name in UPPER_CASE)
https://www.javatpoint.com/java-constant